src/auth/pages/LoginPage.js

```diff
@@ -73,30 +97,45 @@ const LoginPage = ({ globalMessage }) => {
 			{globalMessage && globalMessage.body ?
 				<Alert message={globalMessage.body.message} messageType={globalMessage.body.messageType} /> : <></>}
 
-			{/* <AlertComponent message={message} messageType={messageType} /> */}
-
-			<Form onSubmit={handleSubmit} style={{ width: 'fit-content' }}>
-
-				<Form.Group className="mb-3" controlId="usernameOrEmail">
-					<Form.Label>Username or Email</Form.Label>
-					<Form.Control type="text" name="usernameOrEmail" placeholder="Enter username or email"
-						value={formData.usernameOrEmail} onChange={handleChange} required />
-				</Form.Group>
-
-				<Form.Group className="mb-3" controlId="password">
-					<Form.Label>Password</Form.Label>
-					<Form.Control type="password" name="password" placeholder="Password"
-						value={formData.password} onChange={handleChange} required />
-				</Form.Group>
-
-				<Button onClick={handleClear} variant="primary" className='me-3'>
+			<Alert message={message} messageType={messageType} />
+
+			<form onSubmit={handleSubmit} className="form-custom" style={{ width: 'fit-content' }}>
+
+				<div className="form-group mb-3" controlId="usernameOrEmail">
+					<label className="form-label" for="usernameOrEmail">Username or Email</label>
+					<input
+						type="text"
+						className="form-input"
+						name="usernameOrEmail"
+						id="usernameOrEmail"
+						placeholder="Enter username or email"
+						value={formData.usernameOrEmail}
+						onChange={handleChange}
+						required
+					/>
+					{validationMessage.usernameOrEmail && !validationMessage.isValid && <p className="form-error">{validationMessage.usernameOrEmail}</p>}
+				</div>
+
+				<div className="form-group mb-3" controlId="password">
+					<label className="form-label" for="password">Password</label>
+					<input
+						type="password"
+						className="form-input" name="password" id="password" placeholder="Password"
+						value={formData.password}
+						onChange={handleChange}
+						required
+					/>
+					{validationMessage.password && !validationMessage.isValid && <p className="form-error">{validationMessage.password}</p>}
+				</div>
+
+				<button type='button' className="form-btn me-3" onClick={handleClear} variant="primary">
 					Clear
-				</Button>
+				</button>
 
-				<Button variant="primary" type="submit">
+				<button className="form-btn" disabled={isFormInvalid} variant="primary">
```

Comment:
```html
<button className="form-btn" disabled={!validationMessage.isValid} variant="primary">
```

Reply:
But we can't change the state of the button after the first click, because isValid is changed when running handleSubmit, but in the current situation we can't call this function, because after the first click - isValid = true and it will always be disabled.

Reply:
Attach screenshots

Reply:
![image](https://github.com/NovaServe/healthy-frontend/assets/102480337/d175c0fe-cffd-4f32-ba99-ebee6c75d7b8)